code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -899,26 +899,55 @@
 
 /datum/reagent/medicine/earthsblood //Created by ambrosia gaia plants
 	name = "Earthsblood"
-	description = "Ichor from an extremely powerful plant. Great for restoring wounds, but it's a little heavy on the brain."
+	description = "Ichor from an extremely powerful plant. Great for restoring wounds, but it's a little heavy on the brain. For some strange reason, it also induces temporary pacifism in those who imbibe it and semi-permanent pacifism in those who overdose on it."
 	color = rgb(255, 175, 0)
+	metabolization_rate = 0.4 //Math is based on specific metab rate so we want this to be static AKA if define or medicine metab rate changes, we want this to stay until we can rework calculations.
 	overdose_threshold = 25
 
 /datum/reagent/medicine/earthsblood/on_mob_life(mob/living/carbon/M)
-	M.adjustBruteLoss(-3 * REM, 0)
-	M.adjustFireLoss(-3 * REM, 0)
-	M.adjustOxyLoss(-15 * REM, 0)
-	M.adjustToxLoss(-3 * REM, 0)
-	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2 * REM, 150) //This does, after all, come from ambrosia, and the most powerful ambrosia in existence, at that!
-	M.adjustCloneLoss(-1 * REM, 0)
-	M.adjustStaminaLoss(-30 * REM, 0)
-	M.jitteriness = min(max(0, M.jitteriness + 3), 30)
+	if(current_cycle <= 25) //10u has to be processed before u get into THE FUN ZONE
+		M.adjustBruteLoss(-1 * REM, 0)
+		M.adjustFireLoss(-1 * REM, 0)
+		M.adjustOxyLoss(-0.5 * REM, 0)
+		M.adjustToxLoss(-0.5 * REM, 0)
+		M.adjustCloneLoss(-0.1 * REM, 0)
+		M.adjustStaminaLoss(-0.5 * REM, 0)
+		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 1 * REM, 150) //This does, after all, come from ambrosia, and the most powerful ambrosia in existence, at that!
+	else
+		M.adjustBruteLoss(-5 * REM, 0) //slow to start, but very quick healing once it gets going
+		M.adjustFireLoss(-5 * REM, 0)
+		M.adjustOxyLoss(-3 * REM, 0)
+		M.adjustToxLoss(-3 * REM, 0)
+		M.adjustCloneLoss(-1 * REM, 0)
+		M.adjustStaminaLoss(-3 * REM, 0)
```

Comment:
I would buff this slightly, it does make you a pacifist after all so won't be a combat chem. The rest of the chem is super strong healing overall, so giving this -10 stamina is fine.

Reply:
That's exactly why it DOESN'T need strong healing for stamina.

3% of healing per tick, 5% if brute/burn is fine. It doesn't need to 0-100 you in 3.5

I would consider removing the "baby heals at first" bit though.

Reply:
You know a bit earlier I did think about that time Lexia got fairly close to winning a battle royale as a pacifist and realized how dumb this was. So yeah, leave it.

Reply:
Don't code for the outlier. Unless everyone suddenly becomes as good as lexia black

Reply:
The point is, you can kill people as a pacifist. So yeah it can absolutely be a combat chem, just one side will guaranteed be fighting with non lethals. Until you are cuffed and they gib you somehow.

Reply:
remember, pacifism for the purpose of how it's used ingame does NOT mean you cannot fight.

pacifism just means you can't DIRECTLY use lethals. Nonlethal equipment is still permissible, such as batons/disablers.